lmnet/executor/export.py

```diff
@@ -44,6 +50,7 @@ def _pre_process(raw_image, pre_processor, data_format):
 
 
 def _save_npy(image_path, output_dir, image, raw_image, all_outputs, image_size):
+    shutil.copy(image_path, os.path.join(output_dir))
     shutil.copy(image_path, os.path.join(output_dir, "raw_image.png"))
```

Comment:
Do we need to copy same image to different name? Maybe I missed something.

Reply:
The line `shutil.copy(image_path, os.path.join(output_dir))` copy input image to `out_dir` with reserving input image file name.
The line `shutil.copy(image_path, os.path.join(output_dir, "raw_image.png"))` copy input image to `out_dir` with changing image file name `raw_image.png`.
We need one of them, But I don't know which one should be choose for useful to debug.

Reply:
Thanks, I see, so keeping both is better for debugging purpose, feel that only the `raw_image.png` should be ok though.